modules/ROOT/pages/scopes-and-collections-config.adoc

```diff
@@ -0,0 +1,191 @@
+= Scopes and Collections Configuration for Sync Gateway
+:page-aliases: learn/scopes-and-collections-config.adoc
+ifdef::show_edition[:page-edition: {release}]
+ifdef::prerelease[:page-status: {prerelease}]
+:page-role:
+:page-content: conceptual
+:description: pass:q[How to configure Scopes and Collections for Sync Gateway - Examples with descriptions.]
+
+For a successful configuration of your Couchbase Sync Gateway Scopes and Collections, use this helpful example. 
+First, it's important to remember that buckets can hold multiple scopes, and each scope can contain multiple collections.
+
+When you create a bucket, a `bucket._default._default` element is inherently included, as per the documentation on xref:server/learn/data/scopes-and-collections.adoc#default-scope-and-collection[Default Scopes and Collections] in Couchbase Server.
```

Comment:
I don't think this needs to be named here, but this is referred to as a `keyspace` in N1QL https://docs.couchbase.com/server/current/n1ql/n1ql-intro/sysinfo.html

Keyspaces in sync gateway are similar except they are `<db>.<scope>.<collection>`